### PR TITLE
375 sorting (II) 

### LINKED ISSUE #375: 🚀 [Feature] Sorting
### Sorting

image

- [ ] Latest (newest)
- [ ] Oldest
- [ ] Most Popular (shouts)
- [ ] (optional) Most commented/interactions https://github.com/human-connection/human-connection/issues/265

### Userstory

missing @roschaefer 

### Design & Layout - Alpha

Closed:
image

Open:
image

### Design & Layout - Nitro
missing

### PR DESCRIPTION
**Pull request by ogerly**
## 🍰 Pullrequest
contributions can be sorted 
- createdAt_desc > (Newest)
- createdAt_asc > (Oldest)
- shoutedCount_desc > (Popular)
- commentsCount_desc > (Commented)

the sorting also works with the filter bubble

ezgif com-video-to-gif (3)
### Issues
- fixed #375
  
### Todo
 - [X] Frontend & logic 
-  [X] backend

### CODE REVIEW

**Comment by ogerly** [comment]
@mattwr18  or @Tirokk I would be grateful if you would correct the mistakes. 
i get unfortunately on my system the errors not yet reproduced

**Comment by appinteractive** [comment]
Hey from the UX perspective it makes no sense to have a searchable select there. We should use a dropdown here as it will likely never exceed a couple of entries.

**Comment by mattwr18** [comment]
> Hey from the UX perspective it makes no sense to have a searchable select there. We should use a dropdown here as it will likely never exceed a couple of entries.

I agree with you @appinteractive, but there is not a dropdown from the styleguide which is not searchable, correct? Are you suggesting using just HTML?

**Comment by mattwr18** [comment]
Oh right, we have the `Dropdown.vue` component, :stuck_out_tongue:

File: webapp/locales/de.json [hunk 1]
```diff
@@ -23,6 +23,12 @@
     "bank": "Bankverbindung",
     "germany": "Deutschland"
   },
+  "sorting": {
+    "newest": "Neuste",
+    "oldest": "Älteste",
+    "popular": "Beliebt",
+    "commented": "meist Kommentiert"
```
Comment (Tirokk) [inline webapp/locales/de.json]: I think -> „Meist Kommentiert"

File: webapp/locales/en.json [hunk 2]
```diff
@@ -23,6 +23,12 @@
     "bank": "bank account",
     "germany": "Germany"
   },
+  "sorting": {
+    "newest": "Newest",
+    "oldest": "Oldest",
+    "popular": "Popular",
+    "commented": "most Commented"
```
Comment (Tirokk) [inline webapp/locales/en.json]: I think -> „Most Commented"

File: webapp/pages/index.vue [hunk 3]
```diff
@@ -9,6 +9,17 @@
           @clearSearch="clearSearch"
         />
       </ds-flex-item>
+      <ds-flex-item>
+        <div class="sorting-dropdown">
+          <ds-select
+            v-model="selected"
+            :options="sortingOptions"
+            size="large"
```
Comment (Tirokk) [inline webapp/pages/index.vue]: The typo is a little bit to big, in my eyes.

**Pull request merged**